compiler/passes/checkParsed.cpp

```diff
@@ -544,13 +544,31 @@ static void warnUnstableLeadingUnderscores() {
   if (fWarnUnstable) {
     forv_Vec(DefExpr, def, gDefExprs) {
       const char* name = def->name();
-      
+      Symbol* sym = def->sym;
+      ModuleSymbol* mod = def->getModule();
+      FnSymbol* fn = def->getFunction();
+
       if (name && name[0] == '_' &&
-          def->getModule()->modTag == MOD_USER &&
-          !def->sym->hasFlag(FLAG_TEMP)) {
+          mod && mod->modTag == MOD_USER &&
+          !sym->hasFlag(FLAG_TEMP) &&
+          sym->type != dtMethodToken) {
         USR_WARN(def,
                  "Symbol names with leading underscores (%s) are unstable.", name);
       }
+      if (name &&
+          name[0] == 'c' &&
+          name[1] == 'h' &&
+          name[2] == 'p' &&
+          name[3] == 'l' &&
+          name[4] == '_' &&
```

Comment:
This looks like it will get in trouble for names shorter than five characters.

Reply:
Nope, because C strings are null terminated. So e.g. if the string was `c`, `name[0] == 'c'` would be true, but `name[1] == 'h'` would be false because `name[1]` would be `\0`.

Reply:
(unless you are seeing something that I am not)

Reply:
Oh, right. Nevermind. I think `strncmp` would be nicer for this, but am OK with you leaving it as-is.